test/cookies.spec.js

```diff
@@ -96,7 +97,8 @@ describe('Cookie specs', () => {
           size: 12,
           httpOnly: false,
           secure: false,
-          session: true
+          session: true,
+          priority: 'Medium',
```

Comment:
Since `priority` is not cross-browser yet (I think?), I think we should not expose this property yet. @hanselfmu Could you change `page.cookies()` to remove the `priority` property for now? We can re-enable it once support is more widespread. cc @mjzffr

Reply:
FYI @mjzffr, once this one lands, we plan on cutting a new release.

Reply:
That's right, cookie priority is Chrome-only as far a I know. Thanks for the heads-up.

Reply:
I've removed the 'priority' field from cookies for now.